### DISCUSSION TITLE
Error when loading logging file into behavior tree cpp

### DISCUSSION THREAD

**Discussion by jwallace42** [BehaviorTree/BehaviorTree.CPP#673] [Q&A] [answered]
Hello! 

When loading a logging file(log.fbl) into groot I get the following error. 

```
terminate called after throwing an instance of 'std::out_of_range'
  what():  _Map_base::at

```

I am running groot (version 1) and behavior tree cpp (3.8) on jammy. 

I was wondering if anyone had advise on how to debug/proceed. I will note that when I loaded in the test logger file there was no issues. 

Cheers, 

Josh

**Comment by facontidavide** [comment, marked as answer]
Groot 1 is deprecated and is not supported anymore.
Cheers
